src/sidebar/store/modules/test/selection-test.js

```diff
@@ -295,22 +295,38 @@ describe('sidebar/store/modules/selection', () => {
     it('should an return empty string when there is no focus object', function() {
       assert.equal(store.focusModeUserPrettyName(), '');
     });
+    it('should the username when displayName and userid is missing', function() {
+      // remove once LMS no longer sends username in RPC or config
```

Comment:
If we have a temporary change which should be removed when a specific PR or GitHub issue is resolved, whether that is in our own code or some third party library, it is a useful practice to link to that issue.